### DISCUSSION TITLE
Errors in console after updating "homepage" in package.json

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [mswjs/msw#698] [Q&A] [answered]
Everything with `msw` was working fine up until I added the `homepage` property to my `package.json` file to `"homepage": "car-dashboard",`. I had to update this in order to get it it work on the hosting platform I am using.

However, locally I am now getting these errors:

![image](https://user-images.githubusercontent.com/9777763/115036224-39c6ae00-9ec5-11eb-95f1-50202d88cb5e.png)

Handler:

```
 // GET cars
 rest.get('/v1/cars', (req, res, ctx) => {
   return res(ctx.status(200), ctx.json(mockCars));
 })
```

Any help would be greatly appreciated 🙏

**Comment by kettanaito** [comment, marked as answer]
Hey, @cmacdonnacha. 

Please, have you followed [this troubleshooting guide](https://mswjs.io/docs/getting-started/integrate/browser#using-homepage-property-in-packagejson) when using the custom `homepage` property in `package.json`? Let us know if it helps.

**Reply by cmacdonnacha** [reply]
Thanks @kettanaito missed that in the troubleshooting guide.👍

**Comment by cmacdonnacha** [comment]
For anyone using `create-react-app`, setting `PUBLIC_URL="http://localhost:4000"` in a `.env` file worked and is much cleaner.
